apps/yapms/src/lib/styles/global.css

```diff
@@ -25,3 +25,12 @@ body {
 [data-svelte-typeahead] .svelte-typeahead-list .selected:hover {
 	@apply bg-neutral text-neutral-content font-semibold !important;
 }
+
+#map-div > svg {
+	--auto-border-stroke-width: 0px;
+}
+
+svg > .regions > path {
+	stroke-width: var(--auto-border-stroke-width);
+	stroke: hsla(var(--b1) / var(--tw-bg-opacity, 1));
+}
```

Comment:
Should this also only target the #map-div? I'll do a real review with testing when I get home tonight, just wanted to point it out.

Reply:
Ahh yes good point